### ISSUE TITLE
NVDA 2025.2beta1: Thunderbird is unusable

### ISSUE THREAD

**Issue by paulber19** [nvaccess/nvda#18415]
### Actual behavior:

Start Thunderbird
Press Tab several times
NVDA speaks "unknown".
### Expected behavior:

NVDA must not say unknown
#### NVDA installed/portable/running from source:
NVDA portable
#### NVDA version:
NVDA 2025.2beta1
#### Windows version:
Windows 10 22H2 (10.0.19045.5011) workstation AMD64
#### Name and version of other software in use when reproducing the issue:
thunderbird 128
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No problem with nvda2025.1.2 and nvda alpha-37061,9e668b18 x86

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

[nvda.log](https://github.com/user-attachments/files/21055028/nvda.log)

**Comment by Adriani90** [comment]
Could you please update thunderbird to version 140? I cannot reproduce with NVDA 2025.2 Beta and that version.

**Comment by paulber19** [comment]
No, there is no reason because as I indicated there is no progress with 
NVDA 2025.1.2 and the last alpha.

**Comment by seanbudd** [comment]
@paulber19 - can you please clarify the full version number of thunderbird you are using? There are many versions of "128". Can you please make sure to update to the latest version of 128: [128.12.0esr](https://www.thunderbird.net/en-US/thunderbird/128.12.0esr/releasenotes/). Even if an old version of NVDA used to work with your current version, software evolves and you must keep up to date for it to work. I would also generally encourage an update to 140, which has many accessibility improvement.

@Adriani90 - Thunderbird 128 is the ESR version, so we should support it until it no longer comes under active support.

**Comment by josephsl** [comment]
Hi,
A few things:

* Have you tried with 2025.2 beta 2?
* The Windows version you have is outdated - build 29045.5011 was released in October 2024, and the latest build.revision is 19045.6093.
* Have you tried with latest alpha release (37209)? I ask because you say alpha.37061 works and that was the last alpha build prior to beta 1 being tagged.

Thanks.

**Comment by ruifontes** [comment]
@josephsl, with NVDA Alpha 37209, Windows 11 24H2 26100.4484, Thunderbird 140.0esr (64-bit) with or without add-ons, works alright.

**Comment by josephsl** [comment]
Hi, at least that answers Adriani’s question. The question remains: what is the version of Thunderbird 128 ESR used to reproduce the problem, and can updating to the latest 128 ESR resolve this? Further, can updating to latest Windows 10 cumulative update fix this as well? Thanks.

**Comment by gerald-hartig** [comment]
Closing this issue as it appears that the latest ESR version (140) does not suffer from this behaviour.  v128 is no longer under support as v140 is the new ESR.

**Comment by dan1982code** [comment]
Kind of wondering if OP needs to do NVDA->Tools->Run COM Registration Fixing tool, and restart computer.

**Comment by paulber19** [comment]
The version of Thunderbird is the 128.12.0 ESR 64 bits.
With NVDA 2025.1.2 and alpha-37209, F1A1C809 X86
, NVDA behaves normally. All focus objects are well identified.
Under the 2025.2 beta2 version, objects are announced as "unknown".
With the 140.0 version of Thunderbird, there is no problem with the 3 
versions of NVDA.
